RawSpeed/ArwDecoder.cpp

```diff
@@ -63,6 +63,40 @@ RawImage ArwDecoder::decodeRawInternal() {
       }
 
       return mRaw;
+    } else if (hints.find("srf_format") != hints.end()) {
```

Comment:
If you mean variations of the SRF format that's highly unlikely. The F828 came out in 2003 and all new Sony cameras use ARW. It's just a dead format like many others.